app/Http/Controllers/Api/CtsController.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace App\Http\Controllers\Api;
+
+use App\Repositories\Cts\BookingRepository;
+use Illuminate\Http\Request;
+
+class CtsController
+{
+    private $bookingRepository;
+
+    public function __construct(BookingRepository $bookingRepository)
+    {
+        $this->bookingRepository = $bookingRepository;
+    }
+
+    public function getBookings(Request $request)
```

Comment:
To avoid infinite pagination, I feel it wise here to enforce a maximum 'date in the past' functionality here. Loose suggestion would be you can't retrieve more than 30 days in the past.